docs/dynamic_records.md

```diff
@@ -220,4 +220,22 @@ Sonar check regions (sonar_regions) possible values:
         connect_timeout: 2
         response_timeout: 10
         rapid_recheck: True
-```
\ No newline at end of file
+```
+
+#### [Azure Health Check Options](https://docs.microsoft.com/en-us/azure/traffic-manager/traffic-manager-monitoring#configure-endpoint-monitoring)
+
+| Key | Description | Default |
+|--|--|--|
+| interval_in_seconds          | This value specifies how often an endpoint is checked for its health  from a Traffic Manager probing agent. You can specify two values here:  30 seconds (normal probing) and 10 seconds (fast probing). If no values  are provided, the profile sets to a default value of 30 seconds. Visit  the [Traffic Manager Pricing](https://azure.microsoft.com/pricing/details/traffic-manager) page to learn more about fast probing pricing. | 30      |
+| timeout_in_seconds           | This property specifies the amount of time the Traffic Manager probing  agent should wait before considering a health probe check to an endpoint a failure. If the Probing Interval is set to 30 seconds, then you can  set the Timeout value between 5 and 10 seconds. If no value is  specified, it uses a default value of 10 seconds. If the Probing  Interval is set to 10 seconds, then you can set the Timeout value  between 5 and 9 seconds. If no Timeout value is specified, it uses a  default value of 9 seconds. | 10 or 9 |
+| tolerated_number_of_failures | This value specifies how many failures a Traffic Manager probing agent  tolerates before marking that endpoint as unhealthy. Its value can range between 0 and 9. A value of 0 means a single monitoring failure can  cause that endpoint to be marked as unhealthy. If no value is specified, it uses the default value of 3. | 3       |
+
+```
+---
+  octodns:
+    azuredns:
+      healthcheck:
+        interval_in_seconds: 10
+        timeout_in_seconds: 7
+        tolerated_number_of_failures: 4
```

Comment:
I think terser versions of these are preferable: `interval`, `timeout` (units for both are already in description) and `tolerated_failures`, unless you strongly prefer the current versions.

Reply:
Those names come straight from the [Azure SDK](https://docs.microsoft.com/en-us/python/api/azure-mgmt-trafficmanager/azure.mgmt.trafficmanager.models.monitorconfig?view=azure-python), and I feel like the octoDNS users who will be using these advanced features will appreciate not having to "translate" what name means what in octoDNS and the Azure DNS platform that they're probably already familiar with.

Reply:
Unlikely they will be using octoDNS as well as this part of this SDK for ATM monitors.

Reply:
I doubt anyone would be using it concurrently, but users might be migrating from another tool. The following tools all use the same naming:

- [Terraform](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/traffic_manager_profile)
 - [Azure SDK](https://docs.microsoft.com/en-us/python/api/azure-mgmt-trafficmanager/azure.mgmt.trafficmanager.models.monitorconfig?view=azure-python)
 - [ARM templates](https://docs.microsoft.com/en-us/azure/templates/microsoft.network/trafficmanagerprofiles?tabs=bicep)
 
octoDNS should follow that established convention.

Reply:
They'd still have to read through octoDNS documentation for each field. I agree it would be cognitively easier. Not sure if that should be the primary goal here, but since you still prefer so, I'm ok with this.

Reply:
Thanks!